test/message/test_plan.py

```diff
@@ -287,26 +287,23 @@ def my_plan(data):
     x = th.tensor([-1, 2, 3])
     local_res = my_plan(x)
 
-    kwargs = {"id": "test_plan_worker", "host": "localhost", "port": 8799, "hook": hook}
-    server = start_proc(WebsocketServerWorker, **kwargs)
+    server, remote_proxy = start_remote_worker(id="test_plan_worker", hook=hook)
```

Comment:
port not specified. Is this wanted?
Is there a way to make sure that the process is terminated even when the test fails? I don't see an easy way to implement that right now. It would be easier if the pytest fixture returned a server instance and not the function. (Then you could put it in the teardown code after yield).

If I remember correctly this is one of the reasons for having different port numbers for each test.

Reply:
> Is there a way to make sure that the process is terminated even when the test fails?
I think this is out of the scope of this change. But I agree this would be great to have.

> If I remember correctly this is one of the reasons for having different port numbers for each test.

Yes, indeed. As I said before I'm okay with keeping a default port for tests for now, but in a different PR we can deal with automatic different port allocation and handling fails. What do you think?

Reply:
I agree that it is out of scope to catch port issues. 
However there were no port conflicts possible before this pull request, as we used a different port for each test. So I would propose to stick with separate port numbers.